model/jpa/src/main/java/org/keycloak/connections/jpa/updater/liquibase/custom/CustomKeycloakTask.java

```diff
@@ -73,18 +70,8 @@ public String getConfirmationMessage() {
     @Override
     public void setUp() throws SetupException {
         this.kcSession = ThreadLocalSessionContext.getCurrentSession();
-
         if (this.kcSession == null) {
-            // Probably running Liquibase from maven plugin. Try to create kcSession programmatically
```

Comment:
Just curious, we were using this for what?

Reply:
@pedroigor my guess would be there was a way to run migrations from a maven plugin. That doesn't seem to exist anymore.